static/js/place/overview.tsx

```diff
@@ -36,7 +36,7 @@ class Overview extends React.Component<OverviewPropType> {
       <section className="factoid col-12">
         <div className="row">
           <div className="col-12 col-md-4">
-            <Map dcid={this.props.dcid}></Map>
```

Comment:
should we also delete the Map component, if it's no longer used?

Reply:
agree with deleting Map component if it's no longer used!

Reply:
Update: deleted the old map component.